doc/src/compute_count_type.rst

```diff
@@ -101,18 +101,19 @@ Output info
 """""""""""
 
 This compute calculates a global vector of counts.  If the mode is
-{atom} or {bond} or {angle} or {dihedral} or {improper}, then the
+*atom* or *bond* or *angle* or *dihedral* or *improper*, then the
 vector length is the number of atom types or bond types or angle types
 or dihedral types or improper types, respectively.
 
-If the mode is {bond} this compute also calculates a global scalar
+If the mode is *bond* this compute also calculates a global scalar
 which is the number of broken bonds with type = 0, as explained above.
 
 These values can be used by any command that uses global scalar or
 vector values from a compute as input.  See the :doc:`Howto output
 <Howto_output>` page for an overview of LAMMPS output options.
 
-The scalar and vector values calculated by this compute are "extensive".
+The scalar and vector values returned by this compute are non-negative
```

Comment:
Just say they are both intensive.  I suggest also adding a couple lines up above which illustrate what to do if if type fractions are desired.  I.e. show 3 input script lines which define the compute and a vector-style variable (dividing by natoms or nbonds) and a line for thermo output (with a wildcard).

Reply:
By the way, vector variables need an upper limit when using a wildcard. As far as I know, there is no way to get number of atom types in an input script. If that's right, do you think its worth it to add thermo keywords for number of atom, bond, etc. types?

Reply:
wait, 'extract_setting' might do it...